var/spack/repos/builtin/packages/bufr/package.py

```diff
@@ -90,7 +90,7 @@ def _setup_bufr_environment(self, env, suffix):
         lib = find_libraries(libname + append, root=self.prefix, shared=shared, recursive=True)
         lib_envname = "BUFR_LIB{0}".format(suffix) + append
         inc_envname = "BUFR_INC{0}".format(suffix) + append
-        include_dir = "include_{0}".format(suffix)
+        include_dir = "{0}{1}".format(self.prefix.include.bufr, suffix)
```

Comment:
Do you need an underscore between {0} and {1}? 

This path:
`/Users/stephenh/spack-stack/envs/spack-stack-1.4.0/install/apple-clang/14.0.3/bufr-11.7.1-kzpcigh/include/bufr_4` 

instead of:
`/Users/stephenh/spack-stack/envs/spack-stack-1.4.0/install/apple-clang/14.0.3/bufr-11.7.1-kzpcigh/include/bufr4`

Ie, `bufr_4` instead of `bufr4`

Reply:
Or is the underscore included in `self.prefix.include.bufr`?

Reply:
Good catch, just fixed :)